hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java

```diff
@@ -411,7 +411,7 @@ public boolean isCallTimedOut(Operation op) {
             Address owner = partitionService.getPartitionOwnerOrWait(partition);
 
             if (!memberPartitions.containsKey(owner)) {
-                memberPartitions.put(owner, new ArrayList<Integer>());
+                memberPartitions.put(owner, new ArrayList<>());
```

Comment:
minor: the put-if-absent-then-add idiom can be substituted with more succinct `memberPartitions.computeIfAbsent(owner, k -> new ArrayList<>()).add(partition);`.

Reply:
Converted